src/owSignalSimulator.cpp

```diff
@@ -60,7 +60,7 @@ SignalSimulator::SignalSimulator(const std::string &simFileName,
   pName = PyUnicode_FromString(simFileName.c_str());
   PyObject * temp_bytes = PyUnicode_AsEncodedString(pName, "UTF-8", "strict");
   const char * s = PyBytes_AS_STRING(temp_bytes);
-  s = strdup(s);
+  s = _strdup(s);
```

Comment:
This should not be accepted as the targeted version that sibernetic is building for is older than the deprecation of strdup and the addition of _strdup.
The windows build should instead be targeted to an older version